dispatcher.py

```diff
@@ -6,6 +6,8 @@
 import functools
 import yaml
 
+from sys import platform
```

Comment:
We follow `import foo` for the standard modules across the code.

Reply:
code is removed in updated patch